ci/docker_ci.sh

```diff
@@ -50,6 +50,13 @@ for BUILD_TYPE in "${BUILD_TYPES[@]}"; do
         docker tag "${DOCKER_IMAGE_PREFIX}${BUILD_TYPE}:local" "${DOCKER_IMAGE_PREFIX}${BUILD_TYPE}${IMAGE_POSTFIX}:latest"
         docker push "${DOCKER_IMAGE_PREFIX}${BUILD_TYPE}${IMAGE_POSTFIX}:latest"
     fi
+
+    # Push vX.Y-latest to tag the latest image in a release line
+    if [[ "${AZP_BRANCH}" =~ ${RELEASE_TAG_REGEX} ]]; then
```

Comment:
Even without this change, there will be tagged images pushed for the  "envoyproxy/envoy-google-vrp" image (see https://hub.docker.com/r/sunjayb/envoy-google-vrp/tags for an example where I tested this out). Is that desired?

Reply:
Yep, we want that, thanks.